### PR TITLE
fix 10338 PersistentTopics#getMessageTTL maybe throw nullpointException

### LINKED ISSUE #10338: PersistentTopics#getMessageTTL maybe throw nullpointException
**Describe the bug**
  PersistentTopics#getMessageTTL maybe throw nullpointException

**To Reproduce**
execute  TopicPoliciesTest#testAutoCreationDisabled test 
when topic is not exist , and getMessageTTL will throw nullpointException

### PR DESCRIPTION
**Pull request by baomingyu**
Fixes #10338

Master Issue: #10338

### CODE REVIEW

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java [hunk 1]
```diff
@@ -1515,19 +1515,21 @@ public void removeBacklogQuota(@Suspended final AsyncResponse asyncResponse,
     public Integer getMessageTTL(@PathParam("tenant") String tenant,
                              @PathParam("namespace") String namespace,
                              @PathParam("topic") @Encoded String encodedTopic,
-                             @QueryParam("applied") boolean applied) {
+                             @DefaultValue("false")@QueryParam("applied") boolean applied) {
```
Comment (linlinnn) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java:1518]: The primitive type `boolean`'s default value is false.

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java [hunk 2]
```diff
@@ -1515,19 +1515,21 @@ public void removeBacklogQuota(@Suspended final AsyncResponse asyncResponse,
...
-                .map(TopicPolicies::getMessageTTLInSeconds)
-                .orElseGet(() -> {
-                    if (applied) {
-                        Integer otherLevelTTL = getNamespacePolicies(namespaceName).message_ttl_in_seconds;
-                        return otherLevelTTL == null ? pulsar().getConfiguration().getTtlDurationDefaultInSeconds()
-                                : otherLevelTTL;
-                    }
-                    return null;
-                });
+        Optional<Integer> messageTTL = getTopicPolicies(topicName)
```
Comment (linlinnn) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java:1521]: I'm not sure where the difference after this change

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java (hunk 1 above)
Comment (linlinnn) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java:1518]: maybe you want change like this:
@DefaultValue("false") @QueryParam("applied") Boolean applied

**Comment by 315157973** [comment]
Could you show us the logs of NPE? I didn't see where there will be NPE

**Review by 315157973 — COMMENTED** [review]
Could you add unit tests to cover this case?
I didn’t see any difference after this change

**Comment by MarvinCai** [comment]
> Could you show us the logs of NPE? I didn't see where there will be NPE

+1, orElseGet() should cover the null case, if getTopicPolicies() return null when topic doesn't exist it'll just treated as no value present

**Comment by codelipenghui** [comment]
The pr had no activity for 30 days, mark with Stale label.

**Comment by tisonkun** [comment]
Closed as stale and conflict. Please rebase and resubmit the patch if it's still relevant.

I don't think this patch make effective changes.

**Pull request closed without merging**